manifests/init.pp

```diff
@@ -425,7 +418,7 @@
   }
   if $puppetrun_provider {
     validate_string($puppetrun_provider)
-    if $puppetrun_provider == 'puppetssh' and $puppet_split_config_files {
+    if $puppetrun_provider == 'puppetssh' {
       $real_puppetrun_provider = 'ssh'
       warning('foreman_proxy::puppetrun_provider should be "ssh", not "puppetssh" for 1.12 and above')
```

Comment:
I think so, or even remove it entirely. If master becomes 1.12 only then we probably don't have to worry about compatibility with 1.11 parameter values.